coalib/processes/Processing.py

```diff
@@ -19,6 +19,7 @@
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 from coalib.results.SourceRange import SourceRange
 from coalib.settings.Setting import glob_list
+from coalib.settings.Section import extract_aspects_from_section
```

Comment:
you can merge this with the previous line right
```
from coalib.settings.Section import (
    glob_list, extract_aspects_from_section)
```
this is just a preference though :sweat_smile: